### ISSUE TITLE
How do we prepare test.pkl when we want to evaluate on other domains?

### ISSUE THREAD

**Issue by selva604** [jonmun/MM-SADA-code#6]
Hello.
I want to evaluate it with P02 or P03 in EPIC-Kitchen-55.
How did you create the test.pkl for D1 and D2?

These are not annotation test data of P01 and P08 in the EPIC-Kitchen-55-annotation.

**Comment by jonmun** [comment]
The test pickles were generated from the [EPIC-KITCHEN-55 S1 test set](https://github.com/epic-kitchens/epic-kitchens-55-annotations/blob/master/EPIC_test_s1_timestamps.csv) with actions from a specific participant, e.g. participant_id == "P08", and verb_class in [0...7].

**Comment by jonmun** [comment]
Unfortunately, the verb classes in the EPIC-KITCHENS-55 test sets were not released publically (except P01, P08 and P22 used in this work).

For other participants (e.g. P02 and P03) you can find the verb classes in the [EPIC-KITCHENS-100 validation set](https://raw.githubusercontent.com/epic-kitchens/epic-kitchens-100-annotations/master/EPIC_100_validation.csv) as this contains instances from the S1 test set and additional footage.

Note that EPIC-KITCHENS-100 parses verbs into verb classes slightly differently. Therefore, I'd recommend taking the training instances (train.pkl) from the [EPIC-KITCHENS-100 training set](https://github.com/epic-kitchens/epic-kitchens-100-annotations/blob/master/EPIC_100_train.csv) to overcome this issue.
